package.json

```diff
@@ -121,6 +121,7 @@
     "@storybook/react-native": "^5.3.19",
     "@storybook/react-native-server": "^5.3.19",
     "@turf/random": "^6.0.2",
+    "@types/react-native": "^0.64.12",
```

Comment:
wondering if we should keep this at `0.63.X` just to align with our current version of react native, in case there are any significant differences between the type defs

Reply:
Sure! Ill change it now!